### PR TITLE
[CMR] M3-4530: Linode Landing grid status adjustments and animation

### PR DESCRIPTION
**Pull request by tiffwong**
## Description
List of changes for the Linode Landing grid view:
- If the linode status is not running or offline, add blinking animation
- If the linode status is not running or offline, make it open the `NotificationDrawer` on click
- Remove the underscore from "SHUTTING_DOWN" message
- Add font used for code from the designs 
- Add busy status to linodes being cloned

## Type of Change
- Non breaking change ('update', 'change')

## Note to Reviewers
While I was working on this, I noticed that linodes being cloned are not included in the busy filter and that the content shifts to the left when a modal is opened (also persists on dev) so I'll open another ticket to address these

### CODE REVIEW

File: packages/manager/src/features/linodes/LinodeEntityDetail.tsx [hunk 1]
```diff
@@ -310,11 +322,21 @@ const Header: React.FC<HeaderProps> = props => {
             className={classnames({
               [classes.statusChip]: true,
               [classes.statusRunning]: isRunning,
-              [classes.statusOffline]: isOffline
+              [classes.statusOffline]: isOffline,
+              [classes.statusOther]: isOther,
+
+              statusOther: !['running', 'stopped', 'offline'].includes(
```
Comment (johnwcallahan) [inline packages/manager/src/features/linodes/LinodeEntityDetail.tsx:328]: You've already done this work above, so you could do `statusOther: isOther`.

**Review by johnwcallahan — COMMENTED** [review]
Seems like the chip should blink on the Details page as well. 

Also, when Linodes are being cloned, the Chip is a little off: it doesn't show the full text (e.g. "Cloning to: {linodeName}") and the indicator isn't the right color. 

Not sure we can fit that whole text in the Chip though, but we should at least change the color.

Separately, we'll need to render a notification drawer on Linode Details as well, since currently clicking the chip of a pending status doesn't do anything.

**Review by DevDW — COMMENTED** [review]
Looking good so far; I didn't observe anything else that needs fixing aside from what John identified earlier.

File: packages/manager/src/features/linodes/LinodeEntityDetail.tsx [hunk 2]
```diff
@@ -117,6 +113,10 @@ const LinodeEntityDetail: React.FC<CombinedProps> = props => {
 
   const linodeRegionDisplay = dcDisplayNames[linode.region] ?? null;
 
+  const recentEvent = useSelector((state: any) => {
```
Comment (Jskobos) [inline packages/manager/src/features/linodes/LinodeEntityDetail.tsx:116]: can `import { ApplicationState } from 'src/store'` and replace the `any`

**Review by Jskobos — APPROVED** [review]
Everything's working now 🎉  👍  Here's some pictures of things that imo look a little weird, but they're not blocking for this ticket:

Screen Shot 2020-09-24 at 10 56 44 AM
Screen Shot 2020-09-24 at 10 56 49 AM
Screen Shot 2020-09-24 at 10 56 59 AM
Screen Shot 2020-09-24 at 10 57 56 AM

**Review by WilkinsKa1 — APPROVED** [review]
Changes look good, nice work!

**Pull request merged**